src/qmapshack/gis/search/CSearchLineEdit.h

```diff
@@ -57,6 +59,7 @@ private slots:
     IGisProject * connectedProject = nullptr;
     QTreeWidgetItem * searchItem = nullptr;
 
+    QTimer searchCreationTimer;
```

Comment:
The usual Qt-ish way is to define a pointer. Allocate the timer with new in the ctor. This  class as parent.